_episodes/01-introduction.md

```diff
@@ -69,15 +56,16 @@ teaching skills, but to connect instructors:
 
 > ## Key Readings
 >
-> The best overview of research results in education and learning is
-> Ambrose et al's *[How Learning Works][amazon-hlw]*,
-> which is also an excellent example of what secondary literature ought to look like.
```

Comment:
I rather like pointing out that it's good secondary literature - I wish more fields had overviews like it.